seeds/friday_projects.js

```diff
@@ -0,0 +1,32 @@
+exports.seed = function (knex, Promise) {
+  return Promise.all([
+    knex('projects').del(),
+    knex('photos').del(),
+    knex('team_members').del(),
+    knex('project_teams').del()
+  ])
+    .then(function () {
+      return knex('projects').insert({id: 1, title: 'Project One', description: 'This is project one', repo_url: 'http://github.com/projectone', app_url: 'http://projectone.com', date: '02/08/2016'})
+    })
+    .then(function () {
+      return knex('projects').insert({id: 2, title: 'Project Two', description: 'This is project two', repo_url: 'http://github.com/projecttwo', app_url: 'http://projecttwo.com', date: '03/08/2016'})
```

Comment:
You can insert both of these rows into `projects` by passing the `insert()` method an array instead of an object:

```js
return knex('projects').insert([
  {id: 1, title: 'Project One', description: 'This is project one', ...},
  {id: 2, title: 'Project Two', description: 'This is project two', ...}
])
```

You should do this for all tables.